content/docs/services/cdn-route.md

```diff
@@ -74,6 +76,17 @@ shows the old content after the DNS changes propagate.
 You only need to add a CNAME entry when the `domain`
 field is updated. Refer to ["How to set up DNS"](#how-to-set-up-dns) for guidance.
 
+## External services and applications
+
+To create a custom domain for a service or application that is not a cloud.gov tenant application (such as a public S3 bucket), pass the `origin` option:
+
+```bash
+cf create-service cdn-route cdn-route my-cdn-route \
+    -c '{"domain": "my.example.gov", "origin": "my-app.external-example.gov"}'
+```
+
+There's no need to create a private domain for externally managed services and applications.
```

Comment:
This line is a bit confusing still - we should work a bit on clarifying it (for example - "in other words, you don't need to do the `cf create-domain` step first")